src/messaging/ReliableMessageAnalyticsDelegate.h

```diff
@@ -65,6 +67,9 @@ class ReliableMessageAnalyticsDelegate
         // The outgoing message counter associated with the event. If there is no outgoing message counter
         // this value will be 0.
         uint32_t messageCounter = 0;
+        // If the eventType is kRetransmission, this value will be populated with the number of the
+        // retransmission attempt
```

Comment:
This needs to document what "1" means: the original send, the first retransmission, something else?